initdb.py

```diff
@@ -89,8 +82,8 @@ def create_database(cursor):
         create_sda_db = "CREATE DATABASE {}".format(DB_NAME)
         print("Creating database {}".format(DB_NAME))
         cursor.execute(create_sda_db)
-    except mysql.connector.Error as mysqlError:
-        raise Exception('Failed creating database: {}'.format(mysqlError))
+    except mysql.connector.Error as MySqlError:
```

Comment:
`MySqlError` violates [Google Python Style Guide: Naming](https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming)

make it something like: `mysql_error`